### PR TITLE
What happens when a domain expires

### PR DESCRIPTION
**Pull request by meeunier**
Describes what happens when a domain expires.

### CODE REVIEW

**Comment by meeunier** [comment]
Draft – I've edited it from enom's http://www.enom.com/kb/kb/kb_0402-what-happens-domain-expires.htm?Highlight=expires

Will further tweak.

**Comment by weppos** [comment]
Is this a duplicate of #120? If so, which one should we keep?

PS. @therobot @meeunier in order to qualify for the OMG day, remember we have 24 hours for creating and merging the content. :stuck_out_tongue_winking_eye:

**Comment by meeunier** [comment]
fuckit

**Comment by weppos** [comment]
Generally, the author merges the article after a positive feedback. It generally happens after a few hours or by the end of the day in a specific session.

What I generally do is that I spend the first morning producing content, I then publish it for reviews and I spend a little bit of time reviewing other PRs. Other users are generally following a similar approach, therefore I can expect my content to be reviewed at some point during my afternoon and I merge it before evening.

:us: people are also generally providing early feedback during their day and after that they produce PRs. Likewise, they tend to get feedback by other :us: people and/or they may receive feedback by European people in the early morning of the next day.

As a result, the cycle is that we generally tend to merge all European contributions late in the evening and US contributions late in their evening (and or early in European morning).

I'd say it's responsibility of each author to make sure he/she gets a review in time to merge the article. Sometimes I don't get feedback, what I do is that I tend to ping people directly so that I can get a feedback before leaving.

**Comment by aeden** [comment]
This article references other articles without links. Please link them.

**Comment by meeunier** [comment]
too slow @jacobo , I win.

File: content/articles/what-happens-when-a-domain-expires.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Whois Privacy may cause transfer approval emails to not be delivered
+excerpt: Whois Privacy services often do not properly deliver transfer approval emails, and thus should be disabled before requesting a transfer.
```
Comment (weppos) [inline content/articles/what-happens-when-a-domain-expires.md]: I think this title and excerpt are wrong.

**Pull request merged**